### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by CarLosVegga**
# What's new:

- [x] Created bar Loading component while doing fetch to log or signup user.

https://user-images.githubusercontent.com/60989884/235981580-30791d20-a4d6-44fc-b9bc-c7d4d40aea4b.mov

### CODE REVIEW

File: src/components/Loading/Loading.module.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+div .loading-container {
```
Comment (Carlos24Rz) [inline src/components/Loading/Loading.module.css:2]: Why are we using the div selector? I believe using the className should suffice.

File: src/components/Loading/Loading.module.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    background: var(--blue-gradient);
+    border-radius: 0.5rem;
+    position: relative;
+    left: -15%;
+    animation: barAnimation 1.5s infinite ease-in-out;
+}
+
+@keyframes barAnimation {
+    0% {
+        left: -15%;
```
Comment (Carlos24Rz) [inline src/components/Loading/Loading.module.css:21]: Using left, right, top, bottom animations is not recommended as it is more computationally expensive . Is there a way of doing it with translate?

**Review by Carlos24Rz — CHANGES_REQUESTED** [review]
Just check my comments

**Comment by Di-az** [comment]
LGTM!

**Review by Carlos24Rz — APPROVED** [review]
Great, Nice Work!

**Pull request merged**